File: ServiceDemo/src/main/java/com/servicedemo/server/BinderAImpl.java
package com.servicedemo.server;

import android.os.RemoteException;
import android.util.Log;

public class BinderAImpl extends IBinderA.Stub{
    @Override
    public void methodA(String paramA) throws RemoteException {
        Log.e("lzw","methodA");
    }
}

File: ServiceDemo/src/main/java/com/servicedemo/server/BinderAImpl.kt
package com.servicedemo.server

import android.os.RemoteException
import android.util.Log
import com.servicedemo.server.IBinderA.Stub

class BinderAImpl : Stub() {
    @Throws(RemoteException::class)
    override fun methodA(paramA: String) {
        Log.e("lzw", "methodA $paramA")
    }
}